resources/todomvc/architecture-examples/vue/src/components/TodoComponent.vue

```diff
@@ -2,111 +2,120 @@
     <section class="todoapp">
         <header class="header">
             <h1>Todos</h1>
-            <input type="text" class="new-todo" autofocus autocomplete="off" placeholder="What needs to be done?" v-model="newTodo" @keyup.enter="addTodo" />
+            <input type="text" class="new-todo" autofocus autocomplete="off" placeholder="What needs to be done?"
+                v-model="newTodo" @keyup.enter="addTodo" />
         </header>
         <section class="main" v-show="todos.length" v-cloak>
-            <input type="checkbox" class="toggle-all" v-model="allDone">
+            <div class="toggle-all-container">
+                <input type="checkbox" id="toggle-all-input" class="toggle-all" v-model="allDone" />
+                <label class="toggle-all-label" htmlFor="toggle-all-input">
+                    Toggle All Input
+                </label>
+            </div>
             <ul class="todo-list">
-                <li class="todo" v-for="todo in filteredTodos" :class="{completed : todo.completed, editing : todo === editing }">
+                <li class="todo" v-bind:key="todo.id" v-for="todo in filteredTodos"
```

Comment:
The documentation about `:key` says that it should be just this, not `v-bind:key` => https://vuejs.org/api/built-in-special-attributes.html#key

BTW I see this wasn't here before. I doubt this would explain the performance difference but could it be worth looking?

Reply:
yeah - :key is the shorthand I believe... will adjust
(also already tested, there is no performance gain when I remove it)

Reply:
changed!